### ISSUE TITLE
Same font size for all screenshots for specific device

### ISSUE THREAD

**Issue by cherpake** [fastlane/fastlane#8765]
If possible it would be great if the font size would be the same for all screenshots for specific device.

I know its currently calculated based on the copy for specific screenshot, but what if instead it will be calculated for all screenshots for specific device, and minimal value will be chosen?

All the strings are there, and device resolution is known, so it should be as easy as enumerating through all the strings in the strings file.

**Comment by cherpake** [comment]
<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                           |
| --------------------------- | ----------------------------------------------- |
| OS                          | 10.12.4                                         |
| Ruby                        | 2.2.4                                           |
| Bundler?                    | false                                           |
| Git                         | git version 2.9.2                               |
| Installation Source         | /usr/local/lib/fastlane_lib/bundle/bin/fastlane |
| Host                        | Mac OS X 10.12.4 (16E195)                       |
| Ruby Lib Dir                | /usr/local/lib/fastlane_lib/bundle/lib          |
| OpenSSL Version             | OpenSSL 1.0.2g  1 Mar 2016                      |
| Is contained                | false                                           |
| Is homebrew                 | false                                           |
| Is installed via Fabric.app | false                                           |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/     |
| Xcode Version               | 8.3                                             |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE | en_US.UTF-8 | ✅ |


### fastlane files:

**No Fastfile found**


**No Appfile found**


### fastlane gems

| Gem      | Version | Update-Status      |
| -------- | ------- | ------------------ |
| fastlane | 2.24.0  | 🚫 Update available |


### Loaded fastlane plugins:

**No plugins Loaded**


<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| slack-notifier            | 1.5.1        |
| i18n                      | 0.7.0        |
| json                      | 1.8.1        |
| thread_safe               | 0.3.5        |
| tzinfo                    | 1.2.2        |
| minitest                  | 5.4.3        |
| activesupport             | 4.2.7.1      |
| CFPropertyList            | 2.3.4        |
| claide                    | 1.0.1        |
| colored                   | 1.2          |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.4.2        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.4        |
| terminal-notifier         | 1.7.1        |
| unicode-display_width     | 1.1.2        |
| terminal-table            | 1.7.3        |
| plist                     | 3.2.0        |
| public_suffix             | 2.0.4        |
| addressable               | 2.5.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.5.0        |
| babosa                    | 1.0.2        |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.4        |
| excon                     | 0.54.0       |
| faraday                   | 0.10.0       |
| unf_ext                   | 0.0.7.2      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20161129 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.0.1        |
| gh_inspector              | 1.0.2        |
| uber                      | 0.0.15       |
| representable             | 2.3.0        |
| retriable                 | 2.1.0        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| hurley                    | 0.2          |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.1.0        |
| jwt                       | 1.5.6        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.9.20       |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| dotenv                    | 2.1.1        |
| bundler                   | 1.13.6       |
| faraday_middleware        | 0.10.1       |
</details>


*generated on:* **2017-04-04**
</details>

**Comment by cherpake** [comment]
yep - I gave up on frameit, and wrote my own tool - performance is better cause its using CoreGraphics, and for those like me suffering from light form of OCD and wanting same font size for all screenshots for specific language & device, it provides a solution.
Can be found here: https://github.com/cherpake/fastframe.git
Still work in progress but you can check results here: https://itunes.apple.com/us/app/remote-drive-for-mac-file-browser-player-pro/id1095142462?mt=8

Basically you need to generate screenshots using fastlane snapshot or some other tool, and then run this one in the folder. It will look for device frames and embed screenshot inside the frame, with keyword and title (much like frameit).

**Comment by ohayon** [comment]
Hey @cherpake apologies for the delay here but glad you found a solution that works for you! We would also love a PR to get this functionality inside of _frameit_ if you ever have an opportunity! 🚀

**Comment by cherpake** [comment]
Hi @ohayon, the tool is written in ObjC and not python - I'm not familiar with it :(
